File: source/spring-petclinic-rest/src/main/java/com/spring_petclinic/spring_petclinic_rest/data/VisitRepository.java
package com.spring_petclinic.spring_petclinic_rest.data;

import org.springframework.data.jpa.repository.JpaRepository;
import com.spring_petclinic.spring_petclinic_rest.domain.models.Visit;
import com.spring_petclinic.spring_petclinic_rest.intent.IntentMerge;

/**
 * Spring Data JPA repository for the Visit entity.
 */
@IntentMerge
public interface VisitRepository extends JpaRepository<Visit, Integer> {
}

File: source/nestjs-petclinic-rest/src/repository/visit.repository.ts
import { Repository, EntityRepository } from 'typeorm';
import { CustomRepository } from './../typeorm/typeorm-ex.decorator';
import { Visit } from './../domain/entities/visit.entity';
import { IntentMerge } from './../intent/intent.decorators';

@IntentMerge()
@CustomRepository(Visit)
export class VisitRepository extends Repository<Visit> {
}